src/fuzzfetch/core.py

```diff
@@ -772,7 +775,7 @@ def from_args(
         if args.name is None:
             args.name = obj.get_auto_name()
 
-        final_dir = os.path.realpath(os.path.join(args.out, args.name))
+        final_dir = args.out / args.name
```

Comment:
`realpath` would have resolved symlinks, which I think was on purpose.
```suggestion
        final_dir = (args.out / args.name).resolve()
```